tests/test_xero/test_utils.py

```
@@ -473,3 +473,77 @@ def test_post_payment(mocker, db):
+    xero_connection = XeroConnector(credentials_object=xero_credentials, workspace_id=workspace_id)
+
+    # Mock WorkspaceGeneralSettings
+    with mock.patch('apps.workspaces.models.WorkspaceGeneralSettings.objects.filter') as mock_filter:
+        mock_config = mock.MagicMock()
+        mock_config.import_categories = True
+        mock_config.import_suppliers_as_merchants = True
+        mock_config.import_customers = True
+        mock_filter.return_value.first.return_value = mock_config
+
+        # Test for ACCOUNT
+        assert xero_connection.is_import_enabled('ACCOUNT') == True
```

Ruff (0.11.9)

Avoid equality comparisons to `True`; use `if xero_connection.is_import_enabled('ACCOUNT'):` for truth checks

Replace with `xero_connection.is_import_enabled('ACCOUNT')`

(E712)